File: twoSum/BruteForce.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)):
            compare = target - nums[i]
            for j in range(i+1, len(nums)):
                if nums[j] == compare:
                    return [i, j]

File: 1-twoSum/BruteForce.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize)
{
    int *output = (int *)malloc(sizeof(int) * 2);
    *returnSize = 2;
    int i, j, compare;
    
    for(i = 0; i < numsSize; i++)
    {
        compare = target - nums[i];
        for(j = i+1; j < numsSize; j++)
        {
            if(compare == nums[j])
            {
                output[0] = i;
                output[1] = j;
                
                return output;
            }
        }
    }
    
    return NULL;
}
